### DISCUSSION TITLE
List of oscillators on the Hacktribe firmware

### DISCUSSION THREAD

**Discussion by drmindflip** [bangcorrupt/hacktribe#151] [Q&A] [answered]
Apologies if this is already listed somewhere, but I couldn't find it. Would you mind providing a list of what oscillators are available on the Hacktribe OS by default once it is installed?

**Comment by bangcorrupt** [comment, marked as answer]
All of the oscillators, none of the samples (yet).  I'll gradually expand documentation of [features](https://github.com/bangcorrupt/hacktribe/wiki/Features) in the wiki.
